aws-core/build.gradle.kts

```
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Year-free version, or 
```suggestion
 * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```